Makefile

```diff
@@ -1,6 +1,6 @@
 ARCH := $(shell uname -m)
-MELANGE_DIR ?= ../melange
-MELANGE ?= ${MELANGE_DIR}/melange
+MELANGE ?= $(shell which melange)
+MELANGE_DIR ?= /usr/share/melange
```

Comment:
we should probe for this, because on my desktop setup, it really is `../melange`.